utah-design-system-website/src/react/components/websiteContent/library/components/containers/Accordion/AccordionPropsDocumentation.jsx

```diff
@@ -0,0 +1,122 @@
+              <TableCell>null</TableCell>
+              <TableCell>
+                This css class will be added to the container. This will be added to the main <code>&lt;div&gt;</code> element.
+              </TableCell>
+            </TableRow>
+
+            <TableRow>
+              <TableCell><code className="primary-color">contentClassName</code></TableCell>
+              <TableCell><code>string</code></TableCell>
+              <TableCell>null</TableCell>
+              <TableCell>
+                This css class will be added to the content container - <code>&lt;div&gt;</code> element.
+              </TableCell>
+            </TableRow>
+
+            <TableRow>
+              <TableCell><code className="primary-color">headingLevel</code></TableCell>
+              <TableCell><code>number</code></TableCell>
+              <TableCell>2</TableCell>
+              <TableCell>
+                Dictate the heading level of the accordion header.
+              </TableCell>
+            </TableRow>
+
+            <TableRow>
+              <TableCell><code className="primary-color">headerClassName</code></TableCell>
+              <TableCell><code>string</code></TableCell>
+              <TableCell>null</TableCell>
+              <TableCell>
+                This css class will be added to the accordion header - <code>&lt;button&gt;</code> element.
+              </TableCell>
+            </TableRow>
+
+            <TableRow>
+              <TableCell><code className="primary-color">headerContent</code></TableCell>
+              <TableCell><code>React Node</code></TableCell>
+              <TableCell>null</TableCell>
+              <TableCell>
+                The content of the header. But, you can have it
+                be any element to be rendered inside the element.<br />
+                <b>Note</b>: the content will be rendered inside a heading, adjust accordingly.
+              </TableCell>
+            </TableRow>
+
+            <TableRow>
+              <TableCell><code className="primary-color">id</code></TableCell>
+              <TableCell><code>string</code></TableCell>
+              <TableCell><em>required</em></TableCell>
+              <TableCell>
+                An id to put on the main <code>&lt;div&gt;</code> element. This will also be used for
+                accessibility purposes - will set the <code>aria-controls</code> attribute of the header.
+              </TableCell>
+            </TableRow>
+
+            <TableRow>
+              <TableCell><code className="primary-color">isOpen</code></TableCell>
+              <TableCell>
+                <div className="props-code-wrapper">
+                  <code>true</code>
+                  <span> | </span>
+                  <code>false</code>
+                </div>
+              </TableCell>
+              <TableCell>false</TableCell>
+              <TableCell>
+                This props will determine if the accordion should be rendered close or opened.
```

Comment:
"This prop" instead of "This props".
"rendered closed" instead of "rendered close"